src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -404,6 +416,9 @@ private ElseClauseSyntax Transform( ElseClauseSyntax clause )
 	private ArgumentListSyntax Transform( ArgumentListSyntax argList )
 		=> TransformAll( argList, Transform );
 
+	private TypeArgumentListSyntax Transform ( TypeArgumentListSyntax typeArgList )
+		=> SyntaxFactory.TypeArgumentList( TransformTypes( typeArgList.Arguments ) );

```

Comment:
does `typeArgList` have a `.WithArguments` or something? Just wondering because those `.WithX` buddies preserve syntax trivia (vs creating a new thing with `SyntaxFactory.TypeArgumentList`). There's `.WithTriviaFrom` as well I think for cases where you can't get by with a `.WithX` (either because it doesn't exist or you are changing the node type)